cmake/target_delayload.cmake

```diff
@@ -7,9 +7,9 @@ function(target_delayload target_name)
         message(SEND_ERROR "Delayloading is only supported in MSVC")
     endif()
     foreach(lib ${ARGN})
-        target_link_options(${target_name} PRIVATE /DELAYLOAD:"${lib}")
+        target_link_options(${target_name} PRIVATE /DELAYLOAD:"${lib}" ${DELAYLOAD_IMPLEMENTATION})
```

Comment:
How does ${DELAYLOAD_IMPLEMENTATION} get resolved? I don't see it declared anywhere

Reply:
Yeah, just saw it and removed it :)